sw/airborne/modules/nav/nav_rotorcraft_hybrid.c

```diff
@@ -115,7 +121,7 @@ static void nav_hybrid_goto(struct EnuCoor_f *wp)
     // The time in which it does this is: T = V / a_max
     // The maximum speed at which to fly to still allow arriving with zero
     // speed at the waypoint given maximum deceleration is: V = sqrt(2 * a_max * d)
-    float max_speed_decel2 = fabsf(2.f * dist_to_wp * nav_max_deceleration_sp); // dist_to_wp can only be positive, but just in case
+    float max_speed_decel2 = fabsf(2.f * dist_to_wp * nav_max_deceleration_sp * 0.8f); // dist_to_wp can only be positive, but just in case
```

Comment:
please add at least a comment about this 0.8 value

Reply:
Removed this factor and changed the gain instead.